File: Java/Watermelon.java

import java.util.*;

public class Watermelon{

static void divideMelons(int n)
{
	String str;
	if(n==1 || n==2)
		str="NO";
	else if(n%2==0)
		str="YES";
	else
		str="NO";

	System.out.println(str);

}

public static void main(String [] args)
{
	int n;
	Scanner sc=new Scanner(System.in);
	n=sc.nextInt();
	divideMelons(n);
	
}

}

File: Codeforce/C++/Watermelon.cpp
#include<iostream>
using namespace std;

void divideMelons(int n)
{
	if(n==1 || n==2)
		cout<<"NO";
	else if(n%2==0)
		cout<<"YES";
	else
		cout<<"NO";

}

int main()
{
	int n;
	cin>>n;
	divideMelons(n);
	return 0;
}